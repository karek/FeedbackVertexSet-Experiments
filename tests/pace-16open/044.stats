120,469,56
