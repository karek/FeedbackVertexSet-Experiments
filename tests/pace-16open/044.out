24
38 53 49 111 110 43 102 97 86 74 41 3 60 69 25 52 65 11 80 22 14 78 89 57 
